snap/info.go

```diff
@@ -98,9 +98,10 @@ type Info struct {
 	SideInfo
 
 	// The information in these fields is ephemeral, available only from the store.
-	AnonDownloadURL string
-	DownloadURL     string
-	Prices          map[string]float64 `yaml:"prices,omitempty" json:"prices,omitempty"`
+	AnonDownloadURL  string
+	DownloadURL      string
+	Prices           map[string]float64 `yaml:"prices,omitempty" json:"prices,omitempty"`
+	RequiresPurchase bool               `yaml:"requires-purchase,omitempty" json:"requires-purchase,omitempty"`
```

Comment:
s/RequiresPurchase/MustBuy/

It's okay that the store term is different (and thus JSON), but in snapd we've organized around "buy" rather than "purchase", so let's use it here too.